src/index.html

```diff
@@ -4,9 +4,272 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <a href="#" class="header__logo-link">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <div class="header__content">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item ">
+              <a href="#" class="is-active nav__link">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link"
+              data-qa="nav-hover">Laptops & Computers</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main page__main">
+      <div class="card"
+        data-qa="card">
+        <img class="card__images" src="/src/images/imac.jpeg"
+      alt="imac">
```

Comment:
```suggestion
        <img 
          class="card__images"
          src="/src/images/imac.jpeg"
          alt="imac"
        >
```

Style your divs like this if they have 3 or more props. Change this in all cases